src/vivarium_public_health/risks/effect.py

```diff
@@ -361,6 +361,10 @@ def columns_required(self) -> list[str]:
     # Setup methods #
     #################
 
+    @staticmethod
+    def get_name(risk: EntityString, target: TargetString) -> str:
```

Comment:
We use this to kinda hackily determine which components have NonLogLinearRiskEffects